.github/workflows/asv.yml

```diff
@@ -82,13 +82,39 @@ jobs:
 
     - name: Setup asv
       run: |
-        bash benchmarks/regression/merge_history.sh
-        cp merge_history.txt benchmarks/regression/
-        asv machine --config benchmarks/regression/asv.conf.json --yes
+        asv machine --config benchmarks/regression/asv.conf.json --machine F8s_v2 --os ubuntu-18.04 --arch x86-64 --cpu IntelXeon8168 --num_cpu 8 --ram 16GB
 
     - name: Run benchmarks
       run: |
-        asv run -v --show-stderr --config benchmarks/regression/asv.conf.json HASHFILE:merge_history.txt --cpu-affinity 0-7
+        asv run -v --show-stderr --config benchmarks/regression/asv.conf.json --cpu-affinity 0-7 --machine F8s_v2
+
+    - name: Stash benchmarks results
+      run: |
+        mkdir -p /tmp/asv-results/F8s_v2
+        mv benchmarks/regression/.asv/results/benchmarks.json /tmp/asv-results
```

Comment:
if I don't do this, the `checkout` action below is gonna ditch all the files that are not in the git stage area (not sure why)

Reply:
So the artifact tool is not working I take as you do it by hand?

Reply:
tell me more about this "artifact tool" ?

Doing it by hand because the checkout action below, otherwise, removes all the new files